src/rules/__tests__/valid-expect-in-promise.test.ts

```diff
@@ -325,12 +196,128 @@ ruleTester.run('valid-expect-in-promise', rule, {
         promise.then(() => expect(someThing).toEqual(true));
       });
     `,
-
     `
       it('name of done param does not matter', (nameDoesNotMatter) => {
         const promise = getPromise();
         promise.then(() => expect(someThing).toEqual(true));
       });
     `,
   ],
+  invalid: [
```

Comment:
these are just moved right? no actual change?

Reply:
Yup - every other test has `valid` at the top, & I think the consistency is nice.